### ISSUE TITLE
PDF viewer app update doesn't work

### ISSUE THREAD

**Issue by SkyWheel** [nextcloud/nextcloud-snap#512]
Hi Team,
I have 1 update pending for the PDF viewer app. When I'm clicking update to 1.1.2, it is showing "Updating...", then it is showing that the app has been successfully updated. However, after refreshing the page, the update is still there.

**Comment by SkyWheel** [comment]
So, when I disabled the app and tried to remove it, it is showing Error while removing app. There are no records in logs on the issue.

**Comment by auberginepop** [comment]
There is already an issue for this in the upstream package.
[https://github.com/nextcloud/files_pdfviewer/issues/73](url)

**Comment by PnjDbq** [comment]
Link to issue:  https://github.com/nextcloud/files_pdfviewer/issues/73

TLDR; Snaps are read-only, so the auto-update mechanism can't update core apps (like the PDF viewer).

There's a ticket open to investigate how system apps can be updated inside read-only containers (docker, snap...).  The problem should also resolve itself when the next official build (13.0.2) is released.

**Comment by SkyWheel** [comment]
I think this could be closed.
